frame/dvm-dynamic-fee/src/lib.rs

```diff
@@ -32,25 +36,26 @@ pub mod pallet {
 	// --- darwinia-network ---
 	use darwinia_evm::FeeCalculator;
 
-	pub type InherentType = U256;
+	pub(super) type InherentType = U256;
 
-	pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"dynfee0_";
+	pub(super) const INHERENT_IDENTIFIER: InherentIdentifier = *b"dynfee0_";
```

Comment:
Any idea that you don't want to expose these?

Reply:
I remember that I have searched other pallets in substrate frames that use inherent design and following their doc style.

Reply:
Any example?

Reply:
https://crates.parity.io/pallet_timestamp/index.html